lighthouse-core/formatters/table.js

```diff
@@ -91,6 +94,11 @@ class Table extends Formatter {
         }
 
         switch (key) {
+          case 'previewUrl':
+            if (/^image/.test(result.mimeType)) {
```

Comment:
This feels like we're really doubling down on magic happening in the table formatter/assumptions about other keys that don't seem to have a relationship to the caller. What if `preview` were an object that contained`url` and an optional `mimeType`? Do we have other use cases for non-image previews yet?